CMakeLists.txt

```diff
@@ -77,6 +78,20 @@ find_package(FFTW3 REQUIRED)
 include_directories(${FFTW3_INCLUDES})
 
 ########################################################################
+#Process GROMACS settings
+########################################################################
+
+if(WITH_XTC)
+find_package(GROMACS CONFIG)
+include_directories(${GROMACS_INCLUDE_DIRS})
```

Comment:
This line needs to be move inside the `if(GROMACS_FOUND)`  block, otherwise you might add a `NOTFOUND` variable to the include directories.